### PR TITLE
Stop using deprecated openjdk:8 & 11 images

### PR DESCRIPTION
**Pull request by cap10morgan**
We have to move these versions over to eclipse-temurin too because upstream is deprecating them. I posted a message to the clojure Google Group about this to give folks a heads up. I'll post it in the #docker channel of the Clojurians Slack too.

### CODE REVIEW

File: src/docker_clojure/core.clj [hunk 1]
```diff
@@ -75,16 +75,28 @@
...
+                        (if (= "aarch64" jvm-arch)
+                          "arm64v8"
+                          jvm-arch))
+        platform-flag (if (contains? (or architectures
+                                         cfg/default-architectures)
+                                     host-arch)
+                        nil
+                        (str "--platform=linux/" (first architectures)))
+
+        build-cmd     (remove nil? ["docker" "buildx" "build" "--no-cache"
```
Comment (Quantisan) [inline src/docker_clojure/core.clj:97]: Speaking out loud for sanity check. This is only applicable when we want to build some images locally using this `-main`. Docker Hub reads from the `Dockerfile` directly and run them with their own commands changing our build-cmd to use `buildx` doesn't affect them.

**Review by Quantisan — APPROVED** [review]
thanks, @cap10morgan! and your plan sounds good on making some announcements for the community.

File: src/docker_clojure/core.clj (hunk 1 above)
Comment (cap10morgan) [reply src/docker_clojure/core.clj:97]: That's correct. It uses the manifest and the Dockerfiles, not our build code.

**Comment by cap10morgan** [comment]
https://github.com/docker-library/official-images/pull/12856

**Pull request merged**